### PR TITLE
[Area 1] SSO PDF docs migration

### PR DESCRIPTION
**Pull request by marciocloudflare**
Addresses PCX-4691

### CODE REVIEW

File: content/email-security/_partials/_sso-enforcement.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+_build:
+  publishResources: false
+  render: never
+  list: never
+---
+
+- **None**: Each user can choose SSO, or username and password plus 2FA (this is the recommended setting while testing SSO).
```
Comment (angelampcosta) [inline content/email-security/_partials/_sso-enforcement.md]:
```suggestion
- **None**: This setting allows each user to choose SSO, or username and password plus 2FA (this is the recommended setting while testing SSO).
```

Comment (angelampcosta) [reply content/email-security/_partials/_sso-enforcement.md]: It's just a suggestion to match the format of the other options.

File: content/email-security/account-setup/sso/_index.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+title: SSO integration
+pcx_content_type: how-to
+layout: single
+weight: 4
+---
+
+# SSO integration
+
+For added security and convenience, Cloudflare Area 1 offers support for Security Assertion Markup Language based (SAML-based) single sign-on (SSO) logins. Organizations are able to choose between having users access Area 1 with a username and password plus a two-factor authentication (2FA) code, or using an SSO provider, such as OneLogin or Okta.
```
Comment (angelampcosta) [inline content/email-security/account-setup/sso/_index.md:10]: Should it be "Cloudflare Area 1 offers support for Security Assertion Markup Language based (SAML-based) and single sign-on (SSO) logins."?

File: content/email-security/account-setup/sso/azure.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+# Azure integration guide
+
+This tutorial will walk you through the steps for configuring a non-gallery enterprise application within Azure Active Directory to establish a SAML SSO connection with Area 1.
+
+## 1. Azure Active Directory configuration
+
+1. [Log in to Azure portal](https://portal.azure.com/) and open **Enterprise Applications**.
+
+2. Select **New Application** to create a new application.
```
Comment (angelampcosta) [inline content/email-security/account-setup/sso/azure.md]:
```suggestion
2. Select **New Application**.
```

File: content/email-security/account-setup/sso/azure.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+    |------------------------------------------------|----------------------------------------------------|
+    | **Identifier (Entity ID)**                     | `https://horizon.area1security.com`                |
+    | **Reply URL (Assertion Consumer Service URL)** | `https://horizon.area1security.com/api/users/saml` |
+    | **Sign-On URL**                                | Leave blank                                        |
+    | **Relay State**                                | Leave blank                                        |
+    | **Logout URL**                                 | Leave blank                                        |
+
+8. Select **Save** to exit the Basic SAML Configuration.
+
+9. Select the pencil icon to edit **SAML Signing Certificate**.
```
Comment (angelampcosta) [inline content/email-security/account-setup/sso/azure.md]: I can't see the pencil in the image. It's supposed to be there?

File: content/email-security/account-setup/sso/azure.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+    - **Signing Option**: Select _Sign SAML response_ from the drop-down menu.
+    - **Signing Algorithm**: Select _SHA-1_ from the drop-down menu.
+
+    ![Select Sign SAML response and SHA-1 from the menu](/email-security/static/sso/azure/step9-saml-signing-certificate.png)
+
+10. Select **Save** to exit **SAML Signing Certificate**.
+
+11. Still in the **SAML Signing Certificate** section, find **Federation Metadata XML** and select **Download**. You will need this information for the SSO Configuration in the Area 1 dashboard.
```
Comment (angelampcosta) [inline content/email-security/account-setup/sso/azure.md:57]: "Find" it's a bit generic. Is there a more precise indication where it is?

File: content/email-security/account-setup/sso/azure.md [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+
+    ```txt
+    <?xml version="1.0" encoding="utf-8"?><EntityDescriptor ID="_<YOUR_DESCRIPTIOR_ID>" entityID="https://<YOUR_ENTITY_ID> " xmlns="urn:oasis:names:tc:SAML:2.0:metadata">...
+    ```
+8. Select **Update Settings** to save your configuration.
+
+## 3. Test SSO configuration
+
+After completing both the Azure and Area 1 set ups, you can test your SSO access. 
+For this example, the logo for Area 1 has been updated.
```
Comment (angelampcosta) [inline content/email-security/account-setup/sso/azure.md]:
```suggestion
In this example, the logo for Area 1 has been updated.
```

File: content/email-security/account-setup/sso/azure.md [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+4. Locate the Area 1 Horizon application (or whichever name you gave your application), and select it to initiate your SSO login with Area 1. 
+
+5. If you configured everything correctly, you should be signed in to the Area 1 Portal and redirected to the dashboard.
+
+## Troubleshooting
+
+If you have trouble connecting your Azure account to Area 1, make sure that:
+
+- The user exists in the Area 1 dashboard.
+- The **Identifier** and **Reply URLs** in Azure AD are correct (refer to [step 7 - **Basic SAML Configuration**](#1-azure-active-directory-configuration)).
```
Comment (angelampcosta) [inline content/email-security/account-setup/sso/azure.md]: In the preview it's not taking me to the step.

File: content/email-security/account-setup/sso/generic-sso.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+    title: Generic single sign-on integration guide
+---
+
+# Generic single sign-on integration guide
+
+Below is a generic guide to successfully set up an identity provider based SAML. These options might change depending on your identity provider (IDP). However, make sure you set up the options below or their equivalent.
+
+## 1. Identity Provider SAML setup
+
+1. **Log in** to your SAML provider and access its setup section.
```
Comment (angelampcosta) [inline content/email-security/account-setup/sso/generic-sso.md]:
```suggestion
1. Log in to your SAML provider and access its setup section.
```

File: content/email-security/account-setup/sso/generic-sso.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+    | **Single sign on URL**          | `https://horizon.area1security.com/api/users/saml` |
+    | **Audience URI (SP Entity ID)** | `https://horizon.area1security.com/api/users/saml` |
+    | **Name ID format**              | _Email Address_                                      |
+    | **Application username**        | _Email_                                              |
+    | **Response**                    | _Signed_                                             |
+    | **Assertion signature**         | _Unsigned_                                           |
+    | **Signature Algorithm**         | _RSA_-SHA1                                           |
+    | **Digest Algorithm**            | _SHA1_                                               |
+
+3. In the **Attribute Statements**, add your users to the application. Emails you add here should match emails users already have in the Area 1 dashboard.
```
Comment (angelampcosta) [inline content/email-security/account-setup/sso/generic-sso.md]:
```suggestion
3. In the **Attribute Statements**, add your application users. Emails you add here should match emails users already have in the Area 1 dashboard.
```

**Review by angelampcosta — APPROVED** [review]

File: content/email-security/account-setup/sso/_index.md (hunk 2 above)
Comment (marciocloudflare) [reply content/email-security/account-setup/sso/_index.md:10]: Not really. It's a `SAML-based SSO`, not a `SAML-based and SSO login`

**Pull request merged**